packages/module/patternfly-docs/content/extensions/virtual-assistant/design-guidelines.md

```diff
@@ -0,0 +1,206 @@
+- What are the users’ goals?
+- How in-depth is the assistance the user will need?
+- Does human assistance better serve your users?
+- How is a ChatBot superior to online documentation, contextual support or wizards?
+- What data sources or abilities can this ChatBot leverage to assist your users?
+
+Do not create a ChatBot simply for the sake of novelty.
+
+### When to use a ChatBot 
+
+Use a ChatBot to offer your users on-demand help at any time, including:
+- Technical support and troubleshooting.
+- Product information and documentation.
+- Sales and product recommendations.
+- Training and onboarding.
+- System monitoring and alerts.
+- Community engagement .
+- Feedback collection and surveys.
+
+### When not to use a ChatBot
+
+Do not use a ChatBot when: 
+- A task could be accomplished more efficiently through the UI. 
+- A process is very complex or could take a long time.
+- A real human is needed for sensitive or emotional topics.
+
+## Behavior 
+
+### Accessing a ChatBot
+
+Users can enter a conversation with a ChatBot by clicking on the toggle. Once the ChatBot window opens, the toggle will change to display an "angle down" icon to indicate that clicking the toggle again will minimize the ChatBot. Users can select the toggle at any point in their journey to open and close the ChatBot as needed.
+
+<div class="ws-docs-content-img">
+![2 toggles with open and closed icons.](./img/chatbot-toggle.svg)
+</div>
+
+When there is an unread message from the ChatBot, a notification badge should be placed on the toggle.
+
+<div class="ws-docs-content-img">
+![Toggle with notification badge.](./img/chatbot-toggle-notification.svg)
+</div>
+
+### Using the navigation menu
+
+The ChatBot navigation menu primarily contains a users' conversation history with the ChatBot. Clicking the menu icon opens a side drawer in the ChatBot window. 
+
+By clicking into the navigation menu, users can search through previous conversations and perform additional actions, such as sharing a conversation with others. 
+
+<div class="ws-docs-content-img">
+![Conversation history with an options menu opened on a previous conversation.](./img/conversation-history.svg)
+</div>
+
+### Attaching files 
+
+Using [the attach button](/patternfly-ai/chatbot/overview/design-guidelines#message-bar) in the message bar, users can [attach files](/patternfly-ai/chatbot/messages#file-attachments) to their message to share with the ChatBot. 
+
+The attach button can follow a couple of patterns, including: 
+- Opening the file explorer for a user's operating system
+- Opening a menu with attachment options that are chosen by designers and developers
+
+<div class="ws-docs-content-img">
+![Menu with file attachment options.](./img/attachment-menu.svg)
+</div>
+
+If a message attachment is successful, a label with the file details will be displayed in the message:
```

Comment:
<img width="525" alt="Screenshot 2024-11-25 at 7 10 14 PM" src="https://github.com/user-attachments/assets/ef9e6e12-5db8-4f69-84f3-082be44e308a">

I think it could be helpful to show the above pattern as well - showing the files above the input bar when attached before the message is sent.

Reply:
It displays over the message box content and will sort of hover if you scroll - if you try uploading 2 files in one of the attachment demos, you'll see it.

Reply:
Hm for some reason I can't seem to get that to work. When I try to upload more than one file in the attachment demo, the second file just replaces the first, so there's no error for me. 

But, I also don't see the error message set up in figma yet, so this doesn't need to be a blocker since I'd have to scrap a custom image together, which wouldn't be quite accurate!